### DISCUSSION TITLE
OnPreCull 사용이 가능할까요?

### DISCUSSION THREAD

**Discussion by hanaaple** [naverz/zepeto-studio-kor#606] [5. Q&A]
안녕하세요 카메라 비율 고정을 사용하니 기기의 빈 공간에 UI잔상이 발생하여 OnPreCull을 사용하고 싶은데 사용이 가능한가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재는 지원하지 않으나 지원 가능하도록 준비 중에 있습니다.
감사합니다.

**Comment by hanaaple** [comment]
해결방법이 생겨 공유합니다
OnPreCull 이벤트 함수를 직접 정의해서 만들 순 없지만
변수로 존재하는 함수 자체를 바꿔줄 순 있네요
`Camera.onPreCull = new CameraCallback((camera) => {
  Camera.onPreCull
  //원하는 코드
})`
